### ISSUE TITLE
add role when user add reaction in rewrite version

### ISSUE THREAD

**Issue by TrixCoder** [Rapptz/RoboDanny#44]
async def on_raw_reaction_add(self, payload):
        channel = discord.utils.get(self.guild.text_channels, name='★verify-for-chatting★')
        if not payload.guild_id:
            return

        if payload.channel_id != channel.id:
            return	

        guild = self.get_guild(payload.guild_id)
        member = guild.get_member(payload.user_id)

        if payload.emoji.id != 418966077763223552:
            role = discord.utils.get(guild.roles, name="Verified")
        else:
            return

        await member.add_roles(role, reason='Reaction role')
        await member.send(f'You got verified in {self.guild.name}')
        
    async def on_raw_reaction_remove(self, payload):
        channel = discord.utils.get(self.guild.text_channels, name='★verify-for-chatting★')
        if not payload.guild_id:
            return

        if payload.channel_id != channel.id:
            return	

        guild = self.get_guild(payload.guild_id)
        member = guild.get_member(payload.user_id)

        if payload.emoji.id != 418966077763223552:
            role = discord.utils.get(guild.roles, name="Verified")
        else:
            return

        await member.remove_roles(role, reason='Reaction role')
        await member.send(f'You got unverified in {self.guild.name}')

So basically I am using this code but it is working in my server and when I am trying to use it in another server where my bot is then it doesn't works and not showing me any error in console

**Comment by xGurnoor** [comment]
This seems like an issue with your code/lib and not the bot, RoboDanny. This issue doesn't belong in this Git since it belongs to Danny's bot, RoboDanny and not the lib discord.py. And also since I've used the lib and these functions in general myself, I know there's no issue with the lib and it's more likely your code. Issues like this, in general, belong to the official discord.py Discord server, not in Github, the Github issues are for issues with the library only.

**Comment by Rapptz** [comment]
This has nothing to do with my bot.
